woodwork/column_accessor.py

```diff
@@ -85,13 +89,59 @@ def __eq__(self, other):
             return False
         return self._series.equals(other._series)
 
+    def __getattr__(self, attr):
+        '''
+            If the method is present on the Accessor, uses that method.
+            If the method is present on DataFrame, uses that method.
+        '''
+        if self._schema is None:
+            raise AttributeError("Woodwork not initialized for this Series. Initialize by calling Series.ww.init")
+        if hasattr(self._series, attr):
+            return self._make_series_call(attr)
+        else:
+            raise AttributeError(f"Woodwork has no attribute '{attr}'")
+
     def __repr__(self):
         msg = u"<Series: {} ".format(self._series.name)
         msg += u"(Physical Type = {}) ".format(self._series.dtype)
         msg += u"(Logical Type = {}) ".format(self.logical_type)
         msg += u"(Semantic Tags = {})>".format(self.semantic_tags)
         return msg
 
+    def _make_series_call(self, attr):
+        '''
+        Forwards the requested attribute onto the series object.
+        Intercepts return value, attempting to initialize Woodwork with the current schema
+        when a new Series is returned.
+        Confirms schema is still valid for the original Series.
+        '''
+        series_attr = getattr(self._series, attr)
+
+        if callable(series_attr):
+            def wrapper(*args, **kwargs):
+                # Make Series call and intercept the result
+                result = series_attr(*args, **kwargs)
+
+                # Try to initialize Woodwork with the existing Schema
+                if isinstance(result, pd.Series):
+                    if result.dtype == self._schema['logical_type'].pandas_dtype:
+                        schema = copy.deepcopy(self._schema)
+                        # We don't need to pass dtype from the schema to init
+                        _ = schema.pop('dtype')
```

Comment:
Bit of a nitpick: do we need to set the dummy `_` here? I figure we can just pop without setting the result to anything or, since we know `dtype` is in the schema dict, I'd be fine with doing `del schema['dtype']`

Reply:
Not sure what's preferred...matter of style, I guess. I generally thought it was good to set a dummy to acknowledge that you were intentionally discarding the return, but not a big deal either way. Maybe the `del scheam['dtype']` is better here though. I'll change.